idea/api-gateway/src/kafka/producer.ts

```diff
@@ -10,19 +11,22 @@ async function connect(): Promise<void> {
   await producer.connect();
 }
 
-async function sendByTopic(topic: KAFKA_TOPICS, correlationId: string, params: KafkaParams): Promise<void> {
+async function sendByTopic(topic: KAFKA_TOPICS, params: KafkaParams | string, correlationId?: string): Promise<void> {
   await producer.send({
     topic,
-    messages: [
-      {
-        value: transformToSting(params),
-        headers: {
-          kafka_correlationId: correlationId,
-          kafka_replyTopic: `${topic}.reply`,
-        },
-      },
-    ],
+    messages: [getSendKafkaMessage(topic, params, correlationId)],
   });
 }
 
+function getSendKafkaMessage(topic: string, params: KafkaParams | string, correlationId?: string): Message {
```

Comment:
Weird name

Reply:
@osipov-mit  `getMessageBody` what do you think ?

Reply:
Maybe create instead get?